libtransmission/announcer.cc

```diff
@@ -1567,17 +1569,15 @@ static void scrape_request_delegate(
     }
 }
 
-static void multiscrape(tr_announcer* announcer, tr_ptrArray* tiers)
+static void multiscrape(tr_announcer* announcer, std::vector<tr_tier*> const& tiers)
 {
     size_t request_count = 0;
     time_t const now = tr_time();
-    size_t const tier_count = tr_ptrArraySize(tiers);
     tr_scrape_request requests[MAX_SCRAPES_PER_UPKEEP] = {};
 
     /* batch as many info_hashes into a request as we can */
-    for (size_t i = 0; i < tier_count; ++i)
+    for (auto& tier : tiers)
```

Comment:
Here and elsewhere: if something is a pointer, I'd like to be explicit about it.
```suggestion
    for (auto* tier : tiers)
```

Reply:
fixed ecd14048b7f4a2741646eb10d54d8064a902585a